app/src/main/java/com/github/bsideup/liiklus/service/LiiklusService.java

```diff
@@ -166,7 +149,7 @@
                                                 Flux.from(source.getPublisher())
                                                         .log("partition-" + partition, Level.WARNING, SignalType.ON_ERROR)
                                                         .doFinally(__ -> sourcesByPartition.remove(partition))
-                                                        .transform(Operators.lift(new SubscribeOnlyOnceLifter<Record>()))
+                                                        // TODO .transform(Operators.lift(new SubscribeOnlyOnceLifter<Record>()))
```

Comment:
why todo?

Reply:
this is only a sanity check, was using a class from reactive-grpc before, now it is not available on this classpath, will re-implement later as a follow up